dev/com.ibm.ws.security.common/test/com/ibm/ws/security/common/web/WebUtilsTest.java

```diff
@@ -299,4 +315,521 @@ public void decodeCookie_fullEncodedSpecialChars() {
         }
     }
 
+    @Test
+    public void testStripSecretFromUrlNoChange() {
+        assertNull(WebUtils.stripSecretFromUrl(null, null));
+        assertNull(WebUtils.stripSecretFromUrl(null, "something"));
+
+        String input = "";
+        String expected = input;
+        String secret = "secret";
```

Comment:
Were you intending the value of this variable to actually be `"secret"`, or `"client_secret"`?

Reply:
Since the input is "", it doesn't matter what the secret string is set to in this case.

Reply:
I meant more for the later tests. The `secret` variable is re-used in each of them, but the inputs include `"client_secret"`, not `"secret"`.